### DISCUSSION TITLE
Obtain 20x level from 40x TIFF Image

### DISCUSSION THREAD

**Discussion by alejandrogolfe** [libvips/libvips#2595] [Q&A]
Hi,
I'm having some issues because I need images with levels 40x, 20x,10x,5x. There are some uncompleted images in my dataset. My question is if there is a way to build a 20x level from an image which only have the 40x level and build other levels. 

Thank you for your time!

**Comment by jcupitt** [comment]
Hi @alejandrogolfe, 

You could just regenerate the pyramid, would that work?

Eg.:

```
vips copy missing-levels.tif x.tif[tile,pyramid,compression=jpeg]
```

Now `x.tif` will be a pyramid with all levels present.
